server/app/views/v1/domain_authorizations/_domain_authorization.json.jbuilder

```diff
@@ -0,0 +1,6 @@
+json.domain authorization.domain
```

Comment:
Missing `id` for the new `/v1/domain_authorizations/:id` API

Reply:
done